### DISCUSSION TITLE
discord echo using blackhole + DAW to record system audio [macos catalina]

### DISCUSSION THREAD

**Discussion by xitanyav** [ExistentialAudio/BlackHole#800] [Q&A]
hi all. 

I'm trying to use blackhole to stream audio from a game to discord along with microphone input via the multi-output device + DAW method. However, when I did this I essentially vocoded myself to the game—the people in the stream could only hear the game audio when they heard microphone input [so, only when i talked], and they could also hear an echo of their own voices as well. I don't remember how I achieved this exactly but what should I do instead to prevent that issue from happening?

**Comment by devinroth** [comment]
Sounds like noise cancellation in discord is the issue. And be sure to wear headphones.

Devin

**Reply by xitanyav** [reply]
yep, noise cancellation is definitely the issue, i was just wondering if there was a workaround that wasn't just turning it off.

and the echo's from internally recorded audio. since the application picks up all desktop audio and not just from a specific application, the voices in stream are being picked up by blackhole along with the game audio and everything else, hence the echo.

**Comment by devinroth** [comment]
You should be using headphones when recording.

Devin

**Reply by xitanyav** [reply]
i am! should it not be doing this when i'm wearing headphones?

**Comment by devinroth** [comment]
There should not be any echo when using headphones. You must have audio from the same source coming in multiple times.

Test recording with Audacity.

Devin

**Reply by xitanyav** [reply]
I'm on MacOS catalina, is audacity still having issues with catalina?

**Comment by devinroth** [comment]
I don’t think so.

Devin
